src/core/Accordion/__snapshots__/index.test.tsx.snap

```diff
@@ -23,7 +23,7 @@ exports[`<Accordion /> Test story renders snapshot 1`] = `
       class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
     >
       <div
-        class="css-jbvua0"
+        class="css-1ft6wgl"
```

Comment:
Expected that this CSS class name would change, because we updated its corresponding style component here and throughout this PR